src/python/tensorflow_cloud/core/tests/unit/containerize_test.py

```diff
@@ -30,14 +30,17 @@
     VERSION = "latest"
 
 
+@mock.patch("tensorflow_cloud.core.containerize.requests")
 class TestContainerize(unittest.TestCase):
-    def setup(self):
+    def setup(self, mock_requests):
```

Comment:
Any reason this is not setUp(), which is automatically called for every test methods.

https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp

Reply:
This is something i'll see if i can cleanup across all test files in a separate PR given that we are using unittest framework.